index.js

```diff
@@ -11,7 +12,8 @@ const image = require('./controllers/image')
 
 const db = knex(
     {
-        connectionString: process.env.DATABASE_URL,
+        client: 'postgres',
```

Comment:
Take notes of the changes in this object because I think it was the cause of the errors in heroku